mapit_gb/data/gss-to-local-authority-eng.tsv

```diff
@@ -0,0 +1,445 @@
+E06000016	local-authority-eng:LCE	Leicester
+E10000018	local-authority-eng:LEC	Leicestershire
+E07000063	local-authority-eng:LEE	Lewes
+E09000023	local-authority-eng:LEW	Lewisham
+E07000194	local-authority-eng:LIF	Lichfield
+	local-authority-nir:NIR-B	
+E10000019	local-authority-eng:LIN	Lincolnshire
+	local-authority-nir:NIR-S	
+N09000007	local-authority-nir:LBC	Lisburn and Castlereagh
+E08000012	local-authority-eng:LIV	Liverpool
+E06000032	local-authority-eng:LUT	Luton
+	local-authority-nir:NIR-H	
+E07000110	local-authority-eng:MAI	Maidstone
+E07000074	local-authority-eng:MAL	Maldon
+E07000235	local-authority-eng:MAV	Malvern Hills
+E08000003	local-authority-eng:MAN	Manchester
+E07000174	local-authority-eng:MAS	Mansfield
+E06000035	local-authority-eng:MDW	Medway
+E07000133	local-authority-eng:MEL	Melton
+E07000187	local-authority-eng:MEN	Mendip
+W06000024	local-authority-wls:MTY	Merthyr Tydfil
+E09000024	local-authority-eng:MRT	Merton
+N09000008	local-authority-nir:MEA	Mid and East Antrim
+E07000042	local-authority-eng:MDE	Mid Devon
+E07000203	local-authority-eng:MSU	Mid Suffolk
+E07000228	local-authority-eng:MSS	Mid Sussex
+N09000009	local-authority-nir:MUL	Mid Ulster
+E06000002	local-authority-eng:MDB	Middlesbrough
+S12000019	local-authority-sct:MLN	Midlothian
+E06000042	local-authority-eng:MIK	Milton Keynes
+E07000210	local-authority-eng:MOL	Mole Valley
+W06000021	local-authority-wls:MON	Monmouthshire
+S12000020	local-authority-sct:MRY	Moray
+	local-authority-nir:NIR-E	
+W06000012	local-authority-wls:NTL	Neath Port Talbot
+E07000091	local-authority-eng:NEW	New Forest
+E07000175	local-authority-eng:NEA	Newark and Sherwood
+E08000021	local-authority-eng:NET	Newcastle upon Tyne
+E07000195	local-authority-eng:NEC	Newcastle-under-Lyme
+E09000025	local-authority-eng:NWM	Newham
+W06000022	local-authority-wls:NWP	Newport
+	local-authority-nir:NIR-P	
+N09000010	local-authority-nir:NMD	Newry, Mourne and Down
+	local-authority-nir:NIR-U	
+E10000020	local-authority-eng:NFK	Norfolk
+S12000021	local-authority-sct:NAY	North Ayrshire
+E07000043	local-authority-eng:NDE	North Devon
+E07000050	local-authority-eng:NDO	North Dorset
+	local-authority-nir:NIR-W	
+E07000038	local-authority-eng:NED	North East Derbyshire
+E06000012	local-authority-eng:NEL	North East Lincolnshire
+E07000099	local-authority-eng:NHE	North Hertfordshire
+E07000139	local-authority-eng:NKE	North Kesteven
+S12000044	local-authority-sct:NLK	North Lanarkshire
+E06000013	local-authority-eng:NLN	North Lincolnshire
+E07000147	local-authority-eng:NNO	North Norfolk
+E06000024	local-authority-eng:NSM	North Somerset
+E08000022	local-authority-eng:NTY	North Tyneside
+E07000218	local-authority-eng:NWA	North Warwickshire
+E07000134	local-authority-eng:NWL	North West Leicestershire
+E10000023	local-authority-eng:NYK	North Yorkshire
+E07000154	local-authority-eng:NOR	Northampton
+E10000021	local-authority-eng:NTH	Northamptonshire
+E06000057	local-authority-eng:NBL	Northumberland
```

Comment:
Ah, right, yes, so the example I gave was the same code in areas of two different types (OLF and OLG). Probably easiest for this for now if we just put them on the "latest" GSS codes (ie. what you have already done) and then raise a new ticket to do something about it? This will have a deadline of as soon as ONS issue a new GSS code for a local authority...